test/functional/page_objects/dashboard_page.ts

```diff
@@ -56,7 +56,12 @@ export class DashboardPageObject extends FtrService {
     await this.kibanaServer.savedObjects.cleanStandardList();
     await this.kibanaServer.importExport.load(kibanaIndex);
     await this.kibanaServer.uiSettings.replace({ defaultIndex });
-    await this.common.navigateToApp('dashboard');
+    await this.goToApp();
+  }
+
+  public async goToApp() {
+    const appName = this.config.get('serverless') ? 'dashboards' : 'dashboard';
```

Comment:
Do you know why the app name / the route is different in serverless? IMO it would be preferable to have the same app name across our offerings to avoid confusion and code branches like this.

Reply:
The URL for the dashboard app is `dashboards`. It _should_ work properly if you just use `dashboards` regardless of the offering. I'm looking into this, but my best guess is that the legacy redirect function which redirects from `dashboard` to `dashboards` is just not supported in serverless.

Reply:
Thanks @ThomThomson!